ztp/source-crs/AcceleratorsCatalogSource.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
why adding this ? AND in which PGT (common, group, site) will be used ? There are already many CatalogSource exist under ztp/source-crs that can be overlay

Reply:
Please ignore this, it was picked by mistake. I removed it.